scanpipe/templates/scanpipe/project_list.html

```diff
@@ -21,7 +21,27 @@
             </a>
           {% endif %}
         </div>
-        <a href="{% url 'project_add' %}" class="button is-link">New Project</a>
+        <div>
+          <div class="dropdown is-right is-hoverable">
+            <div class="dropdown-trigger">
+              <button class="button" aria-haspopup="true" aria-controls="dropdown-menu">
+                <span>Sort By</span>
+                <span class="icon is-small">
+                  <i class="fas fa-angle-down" aria-hidden="true"></i>
+                </span>
+              </button>
+            </div>
+            <div class="dropdown-menu" id="dropdown-menu" role="menu">
```

Comment:
`dropdown-menu` is too generic for a unique `id`.

Reply:
what would be better?

`project-list-dropdown`

Reply:
`project-list-dropdown` does not describe what the dropdown is about.
`dropdown-filters` would be better.